config/default.yml

```
@@ -372,3 +372,9 @@ Performance/UriDefaultParser:
   Description: 'Use `URI::DEFAULT_PARSER` instead of `URI::Parser.new`.'
   Enabled: true
   VersionAdded: '0.50'
+
+Performance/UseZipToWrapArrayContents:
+  Description: 'Checks for `.map { |id| [id] }` and suggests replacing it with `.zip`.'
+  Enabled: true
```

```suggestion
  Enabled: pending
```

Cops are not enabled in minor versions